scripts/get_version_from_toml.py

```diff
@@ -0,0 +1,37 @@
+import os
+from pathlib import Path
+import sys
+import toml
+
+
+PYPROJECT_FILE_PATH = "pyproject.toml"
+
+
+def project_root() -> Path:
+    """Root directory of the project."""
+    return Path(os.path.dirname(__file__)).parent
+
+
+def pyproject_file_path() -> Path:
+    """Path to the pyproject.toml."""
+    return project_root() / PYPROJECT_FILE_PATH
+
+
+def get_rasa_version_from_pyproject() -> str:
```

Comment:
could we write some unit tests for this function? if I remember, we have some tests for scripts like this one 🤔